python/lsst/ip/isr/isrTask.py

```diff
@@ -887,17 +930,17 @@ def overscanCorrection(self, exposure, amp):
 
         maskedImage = exposure.getMaskedImage()
         dataView = maskedImage.Factory(maskedImage, amp.getRawDataBBox())
+        overscanImage = maskedImage.Factory(maskedImage, amp.getRawHorizontalOverscanBBox())
```

Comment:
Does this assume overscan is always and forever horizontal? I'm thinking of DECam, which has both horizontal and vertical overscan regions (and its own `isr.py`, of course, so this code doesn't break anything, but I still think it should be as generic as possible).

Reply:
My understanding is that the point of the `overscanCorrection` function is to correct for bias drift as the device is read out. It therefore uses the serial overclock ("horizontal overscan"), because that tracks the bias level as a function of time as the device is read out. I don't believe the parallel overclock ("vertical overscan") is useful for this because it's read out after all the science pixels; to my knowledge we've never used it (for anything). I'm not even sure what use it has unless you have zero bias drift and simply want more pixels to throw into your calculation.